lib/breakers/statsd_plugin.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Breakers
+  class StatsdPlugin
+    def on_outage_begin(outage)
+      StatsD.gauge("api.external_service.#{outage.service.name}.*.up", 0)
```

Comment:
I removed the "resource" after your feedback in https://github.com/department-of-veterans-affairs/devops/pull/909, so the ".*" can be removed.